devel.md

```diff
@@ -0,0 +1,43 @@
+# Development environment for `caasp-init`
```

Comment:
Some of this stuff is redundant with some info from the README.md file. Plus, I don't think is worth giving this information in a separate file, I'd merge this info with the README.md file directly.